develop/components/pages/TablePage.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from "react";
-import { Table, Column, TableRow, PrimaryActionButton, ActionLinkItem, TableHeader, DataItem, FilterItem, FilterProps, EditProps, EditMode } from "../../../src/Table";
+import { Table, Column, TableRow, PrimaryActionButton, ActionLinkItem, TableHeader, DataItem, FilterItem, FilterProps, EditProps, EditMode } from "../../../src/Table/Table";
```

Comment:
the Table folder has an index.ts with `export * from "./Table";` so I'm not sure why you need this change. Does it not work? If not you should remove the index.ts file or fix it there.

Reply:
I replaced the `export * from "./Table";`  with `export { Table } from "./Table";` to make it consistent with the rest of the components

Reply:
Oh ok it's like that in all the components I see. Maybe something we can refactor later.